build/yamls/antrea.yml

```diff
@@ -1467,12 +1468,17 @@ data:
 
     # TLS min version from: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13.
     #tlsMinVersion:
+
+    # Enable namespace label mutator webhook to mutate/validate resources. For example, it
+    # enables the labels mutator which labels all Namespaces with the reserved
+    # "antrea.io/metadata.name = <namespaceName>" label.
```

Comment:
I think the description is too generic, dating back to when you were planning to have a list of webhooks? The "For example" is actually the accurate description, isn't it? There is no validation either, only mutation.

Reply:
yes correct. updated